converter/converter.py

```diff
@@ -15,8 +16,13 @@ def celsius_to_fahrenheit(temperature_in_celsius: float) -> float:
     Temp: 20 C
     20 * (9/5) + 32 = 68 F
     """
-    fahrenheit = round(((temperature_in_celsius * (9 / 5)) + 32), 1)
-    return fahrenheit
+    if temperature_in_celsius < ABSOLUTE_ZERO_CEL:
+        quit(f"Error temperature couldn't be lower than {ABSOLUTE_ZERO_CEL}")
+#  raise Exception(f"Error values is lower than {ABSOLUTE_ZERO_CEL}")
+
+    else:
```

Comment:
@MariaMaximova, if you use `if` statement that checks for exceptional cases and don't continue the method/function execution you can remove `else` part as it will not be executed anyway if exceptional case occurrs.